style.css

```diff
@@ -0,0 +1,104 @@
+/* Основной контейнер */
+.wrapper {
+    margin: 0 auto;
+}
+
+.inputarea {
+    display: block;
+    width: 700px;
+    height: 200px;
+    margin: 10px auto 10px;
+    padding: 10px;
+    font-size: 18px;
+    color: #444444;
+}
+
+.keyboard {
+    width: 704px;
+    margin: 0 auto;
+    padding: 5px 5px 0 0;
+    background: #888888;
+    border: 3px solid #222222;
+    border-radius: 5px;
+}
+
+.language, .description {
+    margin: 10px auto;
+    text-align: center;
+}
+
+.row {
+    clear: both;
+    width: 100%;
+    height: 40px;
+    margin-bottom: 8px;
+}
+
+.keyboard .key {
+    float: left;
+    width: 40px;
+    height: 40px;
+    margin-left: 6px;
+    font-weight: 500;
+    line-height: 40px;
+    text-align: center;
+    color: #ffffff;
+    background: #444444;
+    border-radius: 3px;
+    cursor: pointer;
+    user-select: none;
+    transition: border-radius 0.3s, background-color 0.3s;
+}
+
+.keyboard .key.active {
+    background-color: #2cfd72;
+    border-radius: 50px;
+}
+
+/* Спец кнопки */
+
+.keyboard .Backspace, .keyboard .CapsLock, .keyboard .ShiftLeft {
+    width: 100px;
+}
+
+.keyboard .Enter, .keyboard .ShiftRight {
+    width: 86px;
+}
+
+.keyboard .Tab {
+    width: 50px;
+}
+
+.keyboard .Space {
+    width: 330px;
+}
+
+.keyboard .key:hover {
+    background: #79aa89;

```

Comment:
this rule overriding ".keyboard .key.active" so the active style is never applied if the virtual keyboard used.
another scenario is to click on the button via mouse and move pointer out of it with the key pressed.

Reply:
done